server/api/caseNotesApi.js

```diff
@@ -21,10 +21,12 @@ const caseNotesApiFactory = client => {
 
   const post = (context, url, data) => client.post(context, url, data).then(processResponse(context))
 
+  const put = (context, url, data) => client.put(context, url, data).then(processResponse(context))
```

Comment:
meh... could make this name a bit more routed in the domain, rather than expose the rest-ness of it.
`updateCasenote` (if that's what this does)

Reply:
doesn't need to need to be fixed for this PR though